logs-shipper/src/main/java/jettyjersey/IndexResource.java

```diff
@@ -0,0 +1,69 @@
+package jettyjersey;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.elasticsearch.ElasticsearchException;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Singleton
+@Path("/")
+public class IndexResource {
+
+    private final KafkaProducer<String, String> prod;
+    private static int prodRecNum = 0;
+
+    @Inject
+    public IndexResource(KafkaProducer<String, String> prod) {
+        this.prod = prod;
+    }
+
+    @POST
+    @Path("index")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public String retrieveInfoAndSendToEs(IndexBody indexBody, @Context HttpHeaders httpHeaders) throws IOException {
```

Comment:
Let's return `Response` class instead of String

Reply:
done